liwords-ui/src/gameroom/pool.tsx

```diff
@@ -103,9 +104,9 @@ const Pool = React.memo((props: Props) => {
       placement="bottomRight"
       overlayClassName="format-dropdown"
     >
-      <a href="/" onClick={(e) => e.preventDefault()}>
+      <Link to="/" onClick={(e) => e.preventDefault()}>
         Rearrange
-      </a>
+      </Link>
```

Comment:
hmmm.. will this do the right thing? I wonder if this is the right way to have a link there...